doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
```

Comment:
<p>Some of these stale lock files seem to be related to existence of a directory for previous worker that hasn't cleaned up properly. Can this be avoided? I don't seem to be able to replicate these info messages locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
my bad. Fixed.